src/main/java/models/User.java

```diff
@@ -0,0 +1,77 @@
+package models;
+
+import java.util.Objects;
+
+public class User {
```

Comment:
could we make some of the fields final?

Reply:
- [x] makes name and nickName fields as final, added Lombok @Data for name, nickName and id fields, and @Getter and @Setter for  gameCount field